evmbin/src/main.rs

```diff
@@ -61,6 +61,7 @@ EVM implementation for Parity.
 Usage:
     parity-evm state-test <file> [--json --std-json --only NAME --chain CHAIN]
     parity-evm stats [options]
+    parity-evm stats-jsontests-vm <file> [--folder]
```

Comment:
The name of the command in itself does not seem sufficient to understand what this does. Not sure where the best place to put a proper description of what it does and what the output format is would be, but I think it'd be very useful.